packages/rbac/v2/WorkspacesDelete/index.ts

```diff
@@ -0,0 +1,56 @@
+// @ts-ignore
+import type { AxiosPromise, AxiosInstance, AxiosRequestConfig, Method } from 'axios';
+// @ts-ignore
+import { COLLECTION_FORMATS, RequiredError, AuthTypeEnum, DUMMY_BASE_URL, assertParamExists, setApiKeyToObject, setBasicAuthToObject, setBearerAuthToObject, setOAuthToObject, setSearchParams, serializeDataIfNeeded, toPathString, createRequestFunction } from '@redhat-cloud-services/javascript-clients-shared/dist/common';
+import type { RequestArgs } from '@redhat-cloud-services/javascript-clients-shared/dist/common';
+// @ts-ignore
+import { BaseAPI } from '@redhat-cloud-services/javascript-clients-shared/dist/base';
+import { Configuration } from '@redhat-cloud-services/javascript-clients-shared/dist/configuration';
+
+// @ts-ignore
```

Comment:
These ts-ignores bother me... We may need to investigate this in the future - but I get this is very early stages of the V2 client.